shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -46,7 +49,14 @@
     BridgeExecutorService bridgeExecutorService;
 
     @Inject
-    Tokens tokens;
+    OidcClient client;
+
+    Tokens currentTokens;
+
+    @PostConstruct
+    public void init() {
+        currentTokens = client.getTokens().await().indefinitely();
```

Comment:
@r00ta I think this should also block for 30 seconds and then fail the start-up if we cannot get a token within that time period,  with a clear stack trace.